docs/source/torch.compiler_profiling_torch_compile.rst

```diff
@@ -52,17 +56,24 @@ Here, we observe:
 * CompiledFunction and CompiledFunctionBackward events, which correspond to the dynamo-compiled regions.
 * CPU events at the top, and GPU events at the bottom.
 
-**Flows between CPU and GPU events**
+**Flows between CPU and accelerator events**
 
-Every kernel on the GPU occurs after being launched by code running on the CPU. The profiler can draw connections (i.e. “flows”) between the GPU and CPU events to show which CPU event launched a GPU kernel. This is particularly helpful because, with a few exceptions, GPU kernels are launched asynchronously.
+Every kernel on the GPU/XPU occurs after being launched by code running on the CPU. The profiler can draw connections (i.e. “flows”) between the GPU/XPU and CPU events to show which CPU event launched a GPU/XPU kernel. This is particularly helpful because, with a few exceptions, GPU/XPU kernels are launched asynchronously.
```

Comment:
```suggestion
Every kernel on the accelerator occurs after being launched by code running on the CPU. The profiler can draw connections (i.e. “flows”) between the accelerator and CPU events to show which CPU event launched a accelerator kernel. This is particularly helpful because, with a few exceptions, accelerator kernels are launched asynchronously.
```